src/main/java/calculator/domain/CalculatorNumber.java

```diff
@@ -0,0 +1,21 @@
+package calculator.domain;
+
+public class CalculatorNumber {
+    private String value;
+
+    public CalculatorNumber(String value) {
+        this.value = value;
+    }
+
+    public int parse() {
+        if(isNegativeNumber(value)) {
```

Comment:
IntelliJ IDEA의 코드 자동 정렬 기능(⌥⌘L, Ctrl+Alt+L)을 사용하면 더 깔끔한 코드를 볼 수 있을 거예요.

```suggestion
        if (isNegativeNumber(value)) {
```

Reply:
수정 하였습니다 !